DotnetIntern2021/Basic/Exercise13.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+namespace DotnetIntern2021.Basic
+{
+    public class Exercise13
+    {
+        public List<String> restoreIpAddresses(String s)
+        {
+            var result = new List<String>();
+            for (int a = 0; a <= 255; a++)
+            {
+                var ip1 = a.ToString();
+                if (s.StartsWith(ip1))
+                    for (int b = 0; b <= 255; b++)
+                    {
+                        var ip2 = ip1 + b.ToString();
+                        if (s.StartsWith(ip2))

```

Comment:
Câu lệnh if chỉ nên viết ở 2 dạng

Dạng simple:

```cs
if (điều kiện) câu_lệnh_đơn;
```

Dạng phức tạp
```cs
if (điều kiện) 
{

}
```

```suggestion
                        if (s.StartsWith(ip2))
                        {
                        ....
                        }

```